### PR TITLE
Add examples of how operators work

### PR DESCRIPTION
**Pull request by pattra**
Adding some examples of how the operators should work. Purpose is to hopefully lessen any confusion incoming candidates may feel about operator behaviors.

prime: @salsify/frontend-developers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,3 +60,19 @@ Operators define the relationship between properties and property values. Certai
...
+
+| Operator | Example Property | Example Value | Expected Output |
+| -------- | ---------------- | ------------------- | --------------- |
+| Equals | `Name` | `Headphones` | Products where `Name` is exactly `Headphones` |
+| Is greater than | `Price` | `20` | Products where the `Price` > `20` |
+| Is less than | `Price` | `20` | Products where `Price` < `20` |
+| Has any value | `Description` | --- | Products where `Description` is defined/is NOT null |
+| Has no value | `Description` | --- | Products where the `Description` is not defined/IS null |
+| Is any of | `Name` | `Headphones, Keys` | Products where the Name is either exactly `Headphones` OR exactly `Keys` |
+| Contains | `Name` | `phone` | Products where the Name string CONTAINS `phone` (e.g. `Headphones`, `Telephone`) |
```
Comment (deverstalmage) [inline README.md]: does `Phone` contain `phone`?

Comment (pattra) [reply README.md]: good question. i would probably say yes? at least for this particular operator i care less/not at all(??) about exact casing matches.

**Review by deverstalmage — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (kehphin) [reply README.md]: So then maybe the verbiage would be something like: `Products where the Name string CONTAINS phone, case insensitive (e.g. Headphones, Telephone, Phone)`

Comment (kehphin) [reply README.md]: This also differentiates `contains` from `equals` a bit more as well.

**Pull request merged**